Enterprise-CMCS/macpro-mdct-mfp
# [MDCT-2736] - Add Form component to MFP

### Description

![Screenshot 2023-08-09 at 11 10 02 PM](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/19439679/d0124a90-e6b0-4a38-b86a-5e09411a6d59)


### Related ticket(s)


MDCT-2736

---
### How to test

Add 
```
<Route
  path="/standard"
  element={<StandardReportPage route={mockStandardReportPageJson} />}
/>
```
to AppRoutes and then navigate to /standard. You'll be able to see the form component with (very) loose validation running!

---
### Author checklist


---
